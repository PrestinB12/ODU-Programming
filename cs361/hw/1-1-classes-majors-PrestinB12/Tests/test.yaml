test:
  timelimit: 10
  launch: ./listMajor
  stderr: false
dueDate: 09/07/2023
latePenalties: [10, 20, 100]
build:
  command: make
  weight: 1
  studentFiles:
    include: ["readingList.h", "readingList.cpp"]
  instructorFiles:
    exclude: ["readingList.h", "readingList.cpp"]
