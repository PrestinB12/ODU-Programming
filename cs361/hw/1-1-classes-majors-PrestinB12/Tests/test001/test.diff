14,15c14
<    cs324 (cs374,cs374,cs374)
< **Memory leaks detected: 90
---
>    cs324 (cs216,cs104,cs374)
