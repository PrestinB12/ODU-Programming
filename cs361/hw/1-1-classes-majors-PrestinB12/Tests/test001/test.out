Semester 1
   cs103 ()
   math241 ()
Semester 2
   cs104 (cs103)
Semester 3
   cs201 (cs104)
   cs216 (cs104,math241)
Semester 4
   cs351 (cs201)
   cs374 (cs201)
   cs381 (cs216)
Semester 5
   cs324 (cs374,cs374,cs374)
**Memory leaks detected: 90
